module/Accounting/public/js/accounting.js

```diff
@@ -1,6 +1,33 @@
 var Accounting = function()
 {
-	this.bindEventsOn(); 
+	var defaultPageSize = 10,
```

Comment:
Ho aggiunto questo valore di default in questa classe perchè, passando dalla visualizzazione del MyAccount all'OrganizationAccount, e quindi cambiando vista, dovevo resettare il valore del pageSize, altrimenti sarebbe rimasto lo stesso. Ho riportato poi questo valore anche negli altri moduli (people e task-management), anche se lì non è effettivamente molto utile; la rimuovo dove non necessario.